app/code/Magento/Cron/Model/ResourceModel/Schedule.php

```diff
@@ -69,8 +69,8 @@ public function trySetJobUniqueStatusAtomic($scheduleId, $newStatus, $currentSta
         // this condition added to avoid cron jobs locking after incorrect termination of running job
         $match = $connection->quoteInto(
             'existing.job_code = current.job_code ' .
-            'AND (existing.executed_at > UTC_TIMESTAMP() - INTERVAL 1 DAY OR existing.executed_at IS NULL) ' .
-            'AND existing.status = ?',
+            'AND existing.status = ? ' .
+            'AND (existing.executed_at > UTC_TIMESTAMP() - INTERVAL 1 DAY OR existing.executed_at IS NULL)',
```

Comment:
Moving to same order as index so easier to verify at a glance possible indexes

Reply:
after back port from 2.3 will be delivered in 2.4 this changes will be irrelevant
Magento will use looks instead status field to  avoid simultaneous running jobs

Reply:
I couldn't see any change in this area in 2.4